onnxruntime/python/tools/tensorrt/perf/benchmark.py

```diff
@@ -45,8 +45,8 @@
 # latency gain headers 
 trt_cuda_gain = 'TRT_CUDA_gain(%)'
 trt_cuda_fp16_gain = 'TRT_CUDA_fp16_gain(%)'
-trt_native_gain = 'EP_Native_TRT_gain(%)'
-trt_native_fp16_gain = 'EP_Native_TRT_fp16_gain(%)'
+trt_native_gain = 'EP_Standalone_TRT_gain(%)'
```

Comment:
this new naming feels a little bit inconsistent with headers such as 'TRT_CUDA_gain(%)'